README.md

```diff
@@ -1,47 +1,15 @@
 ## Description
 
-Interface (driver) software, including ROS node, for inertial sensors compatible with the [Microstrain Communication Library (MSCL)](https://github.com/LORD-MicroStrain/MSCL).
+Interface (driver) software, including ROS node, for the [Microstrain](https://microstrain.com) line of inertial sensors from [Parker](http://parker.com), developed in Williston, VT.
```

Comment:
Something I've been trying to be more consistent with (but obviously wasn't before here) is capitalizing the S in MicroStrain. Same thing on the next line.